examples/k8s-scaleway/main.tf

```diff
@@ -0,0 +1,147 @@
+# ###########################
+# INFRA + K8s PHASE
+# ###########################
+module "scaleway" {
+  source = "git@github.com:camptocamp/devops-stack-module-cluster-scaleway.git"
+
+  cluster_name        = var.cluster_name
+  cluster_description = var.cluster_description
+  cluster_tags        = var.cluster_tags
+  kubernetes_version  = var.kubernetes_version
+  lb_name             = var.lb_name
+  lb_type             = var.lb_type
+  zone                = var.zone
+  node_pools          = var.node_pools
+}
+
+# ###########################
+# BOOTSPRAP APPLICATION PHASE
+# ###########################
+
+module "argocd_bootstrap" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-argocd.git//bootstrap?ref=v2.0.0"
+}
+
+module "ingress_controller" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-traefik.git?ref=v1.2.3"
+
+  cluster_name           = var.cluster_name
+  argocd_namespace       = module.argocd_bootstrap.argocd_namespace
+  base_domain            = module.scaleway.base_domain
+  enable_service_monitor = var.ingress_enable_service_monitor
+
+  helm_values = [{
+    traefik = {
+      service = {
+        type = "LoadBalancer"
+        annotations = {
+          "service.beta.kubernetes.io/scw-loadbalancer-id" = module.scaleway.lb_id
+        }
+      }
+    }
+  }]
+
+  dependency_ids = {
+    argocd = module.argocd_bootstrap.id
+  }
+}
+
+module "cert-manager" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-cert-manager.git//self-signed?ref=v4.0.2"
```

Comment:
Why are you using the self-signed variant? It would be better to use Let's Encrypt (which you can set to staging for development purposes). I don't recall right now if there is any special setup that you would need to do to Scaleway.